db/fixtures/miq_product_features.yml

```diff
@@ -3543,6 +3543,44 @@
       :feature_type: control
       :identifier: middleware_datasource_tag
 
+# Middleware Messaging
+- :name: Middleware Messaging
+  :description: Everything under Middleware Messaging
+  :feature_type: node
+  :identifier: middleware_messaging
+  :children:
+  - :name: View
+    :description: View Middleware Messaging
+    :feature_type: view
+    :identifier: middleware_messaging_view
+    :children:
+    - :name: List
+      :description: Display Lists of Middleware Messaging
+      :feature_type: view
+      :identifier: middleware_messaging_show_list
+    - :name: Show
+      :description: Display Individual Middleware Messaging
+      :feature_type: view
+      :identifier: middleware_messaging_show
+  - :name: Modify
+    :description: Modify Middleware Messaging
+    :feature_type: admin
+    :identifier: middleware_messaging_admin
+    :children:
+    - :name: Remove
```

Comment:
Certainly not the subsystem. And I don't even know about individual destinations. As we don't add them, I think we als don't delete them - wdyt?